### PR TITLE
AlloyDB quickstart

### PR DESCRIPTION
**Pull request by 4demos**
# Summary

A new quickstart for GCP AlloyDB

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x ] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [?] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/gcp/gcp-alloydb/config.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+id:
```
Comment (jcountsNR) [inline quickstarts/gcp/gcp-alloydb/config.yml:1]:
```suggestion

```

Comment (jcountsNR) [reply quickstarts/gcp/gcp-alloydb/config.yml:1]: This should be completely blank

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+          "visualization": {
+            "id": "viz.table"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:27]:
```suggestion
                "accountId": 0,
```

Comment (jcountsNR) [reply dashboards/gcp-alloydb/gcp-alloydb.json:27]: these should all be changed to the suggestion.

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:57]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:90]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:123]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:156]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 7]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:190]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 8]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:224]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 9]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:257]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 10]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:291]:
```suggestion
                "accountId": 0,
```

File: dashboards/gcp-alloydb/gcp-alloydb.json [hunk 11]
```diff
@@ -0,0 +1,341 @@
...
```
Comment (jcountsNR) [inline dashboards/gcp-alloydb/gcp-alloydb.json:325]:
```suggestion
                "accountId": 0,
```

File: quickstarts/gcp/gcp-alloydb/config.yml [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+title: AlloyDB
+documentation:
+  - name: AlloyDB integration docs
+    description: Monitor AlloyDB by connecting GCP to New Relic.
+    url: https://docs.newrelic.com/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-alloydb-monitoring-integration/
+keywords:
+  - gcp
+  - google cloud platform
+  - alloydb
+installPlans:
```
Comment (brammerl) [inline quickstarts/gcp/gcp-alloydb/config.yml:31]: Please reference a data source instead. The data source equivalent for this install plans is: google-cloud-platform

Comment (4demos) [reply quickstarts/gcp/gcp-alloydb/config.yml:31]: @brammerl do you mean replace `installPlans:` with dataSource:` ?

Comment (brammerl) [reply quickstarts/gcp/gcp-alloydb/config.yml:31]: yes.

Comment (4demos) [reply quickstarts/gcp/gcp-alloydb/config.yml]: done

**Review by seemantk — APPROVED** [review]

**Pull request merged**